### PR TITLE
vote create

### PR DESCRIPTION
**Pull request by rankitranjan**
https://github.com/brandslisten/bl-core-logic/pull/316l

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -2909,6 +2909,28 @@ paths:
...
             $ref: '#/definitions/ParticipationRequestComments'
+  '/votes':
+    post:
+      x-swagger-router-controller: mock
+      description: >
+        You can vote object when you know it's voteable UUID. It could uuid of (topics, answers and widgets)
+      parameters:
+        - $ref: '#/parameters/X-User-Token'
+        - name: uuid
+          in: path
```
Comment (Xtimesx) [inline swagger.yaml:2920]: isn't that in the body of the request?

File: swagger.yaml [hunk 2]
```diff
@@ -2909,6 +2909,28 @@ paths:
...
+          required: true
+          type: string
+          format: uuid
+      produces:
+        - application/json
+      tags:
+        - Vote
+      responses:
+        '200':
+          description: Succssful response
```
Comment (Xtimesx) [inline swagger.yaml:2931]: shouldn't this create a vote response?

File: swagger.yaml (hunk 1 above)
Comment (rankitranjan) [reply swagger.yaml]: done

File: swagger.yaml (hunk 2 above)
Comment (rankitranjan) [reply swagger.yaml:2931]: done

**Pull request merged**